cooltools/cli/sample.py

```diff
@@ -5,6 +5,7 @@
 
 from . import cli
 from .. import api
+from ..lib.common import pool_decorator
```

Comment:
same question-- maybe more clear if the pool decorator is always in API rather than CLI version of the user-facing function?